src/aks-preview/azext_aks_preview/aks_diagnostics/commands.py

```diff
@@ -0,0 +1,452 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import colorama  # pylint: disable=import-error
+import datetime
+import json
+import os
+import subprocess
+import tempfile
+import time
+
+from azure.cli.core.commands.client_factory import get_mgmt_service_client, get_subscription_id
+from knack.log import get_logger
+from knack.prompting import prompt_y_n
+from knack.util import CLIError
+from msrestazure.azure_exceptions import CloudError
+from tabulate import tabulate  # pylint: disable=import-error
```

Comment:
The `commands` file is usually used to implement `load_command_table()` to register commands. Please do not implement custom methods in it

Reply:
As suggested [here](https://github.com/Azure/azure-cli-extensions/pull/4933#issuecomment-1148131488), I moved this file out of the subdirectory and renamed it to `aks_diagnostics.py`